### PR TITLE
Task/32 user can select search results

### PR DESCRIPTION
**Pull request by Techies-John**
## Problem/Issue:
- #32 

1. Users cannot click on the suggested results.
2. Users cannot navigate to the search results page when they press `Enter`.
3. The components are disorganized.

## Proposed Solution:
1. Implement functionality that redirects users to a specific paper by accessing its `id`.
2. Ensure that when users press `Enter`, only the suggestions provided by the search engine are displayed.
3. Restructure and rename components.

**Note: As I progressed, I encountered several bugs and I've created separate tasks to address them.**

### CODE REVIEW

**Review by nonsensenv — DISMISSED** [review]

**Review by nonsensenv — DISMISSED** [review]

**Review by sggenDebugs — CHANGES_REQUESTED** [review]
we'll talk about the db, services, use-case soon if we both agree on it or not before i approve this.

File: components/Welcome/Page.vue [hunk 1]
```diff
@@ -28,6 +28,10 @@ onMounted(() => {
 
 
 <style scoped>
+body {
+  margin: 0;
+}
+
```
[the remark this answers is not in the document]
Comment (Techies-John) [reply components/Welcome/Page.vue:34]: Okay I'll remove it

File: components/login/Page.vue [hunk 2]
```diff
@@ -103,8 +89,8 @@ const togglePassword = () => {
 
 
 <style scoped>
-
-body,html {
+body,
+html {
```
[the remark this answers is not in the document]
Comment (Techies-John) [reply components/login/Page.vue:93]: Okay I'll remove it

[the remark this answers is not in the document]
Comment (Techies-John) [reply components/login/Page.vue:93]: Okay I'll remove it

**Review by sggenDebugs — APPROVED** [review]
approved ching chong

**Review by nonsensenv — APPROVED** [review]
next time please put version (e.g  alpha build v1.0.5)

**Comment by sggenDebugs** [comment]
Picture is not display correctly. Take note thanks ching chong

**Comment by Techies-John** [comment]
> next time please put version (e.g  alpha build v1.0.5)

Response (Version 1.0.0)
- Okay brad

**Pull request closed without merging**